backend/curriculum_tracking/serializers.py

```diff
@@ -611,3 +611,19 @@ def get_oldest_card_in_review_time(self, instance):
 
     def get_total_cards_in_review(self, instance):
         return self._get_review_agile_cards(instance).count()
+
+
+class ReviewTrustSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.ReviewTrust
+        fields = [
+            "id",
+            "content_item",
+            "user",
```

Comment:
flavours are also relevent. Check what other endpoints expose flavours, you can use the same pattern.

Eg: A person might be trusted for the Animals project, but only in Python (not in Javascript or Java)